pkg/kubectl/cmd/logs.go

```diff
@@ -54,13 +54,19 @@ var (
 		kubectl logs --tail=20 nginx
 
 		# Show all logs from pod nginx written in the last hour
-		kubectl logs --since=1h nginx`)
+		kubectl logs --since=1h nginx
+
+		# Return snapshot logs from first container of a job named hello
+		kubectl logs job/hello
+
+		# Return snapshot logs from from container hello-1 of a job named hello
+		kubectl logs job/hello -c hello-1`)
```

Comment:
Provide an example with a different resource since you already have one with jobs above